exercises/6_vehicle_subclass.rb

```diff
@@ -0,0 +1,38 @@
+class Vehicle
+  attr_accessor :price
+  
+  def initialize(name, price)
+    @name = name
+    @price = price
+  end
+end
+
+class Bike < Vehicle
+  def initialize(name, price, dealer)
+    super(name, price)
+    @dealer = dealer
+    @percent_price_increase = 0.0
```

Comment:
Do we need to initiate ppi with 0.0? Can't we just use attr_accessor here..? Think once

Reply:
yes we can use that also but there is a separate method to do so according to the question